core/src/test/scala/com/databricks/labs/remorph/parsers/snowflake/SnowflakeCommandBuilderSpec.scala

```diff
@@ -0,0 +1,97 @@
+package com.databricks.labs.remorph.parsers.snowflake
+
+import com.databricks.labs.remorph.parsers.intermediate._
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import org.scalatestplus.mockito.MockitoSugar
+
+class SnowflakeCommandBuilderSpec
+    extends AnyWordSpec
+    with SnowflakeParserTestCommon
+    with Matchers
+    with MockitoSugar
+    with IRHelpers {
+
+  override protected def astBuilder: SnowflakeCommandBuilder =
+    new SnowflakeCommandBuilder
+
+  "translate Declare to CreateVariable Expression" in {
+    example(
+      "X NUMBER DEFAULT 0;",
+      _.declareStatement(),
+      CreateVariable(
+        name = Id("X"),
+        dataType = DecimalType(None, None),
+        defaultExpr = Some(Literal(short = Some(0))),
+        replace = false))
+
+    example(
+      "select_statement VARCHAR;",
+      _.declareStatement(),
+      CreateVariable(name = Id("select_statement"), dataType = VarCharType(None), defaultExpr = None, replace = false))
+
+    example(
+      "price NUMBER(13,2) DEFAULT 111.50;",
+      _.declareStatement(),
+      CreateVariable(
+        name = Id("price"),
+        dataType = DecimalType(Some(13), Some(2)),
+        defaultExpr = Some(Literal(float = Some(111.5f))),
+        replace = false))
+
+    example(
+      "query_statement RESULTSET := (select col1 from some_table);",
```

Comment:
```suggestion
      "query_statement RESULTSET := (SELECT col1 FROM some_table);",
```
can we consistently use UPPERCASE for keywords everywhere?